downloads/swan-lake-release-notes/swan-lake-alpha/RELEASE_NOTE.md

```diff
@@ -0,0 +1,1562 @@
+
+    ```ballerina
+    import ballerina/udp;
+
+    service on new udp:Listener(48829) {
+
+    remote function onBytes(readonly & byte[] data, udp:Caller caller) returns (readonly & byte[])|udp:Error? {
+            return data;
+        }
+
+    remote function onError(readonly & udp:Error err) {
+        }
+    }
+    ```
+
+###### WebSubHub 
+
+- This module contains the API specifications and implementations related to the WebSub Hub, WebSub Hub Client and WebSub Publisher.
+- This is an inter-dependent module for the `websub` module.
+
+**Hub Implementation**
+
+- The default `websub:Hub` implementation has been removed and language-specific API abstraction is defined in the `websubhub:Service`.
+- Updated the WebSub Hub sample implementation to comply with the new API specification as follows.
+
+    ```ballerina
+    listener websubhub:Listener hubListener = new (9001);
+
+    service / websubhubon hubListener {
+
+    remote function onRegisterTopic(TopicRegistration message) returns TopicRegistrationSuccess|TopicRegistrationError {
+        // implement action to execute on topic-registration
+        }
+
+    remote function onDeregisterTopic(TopicDeregistration message) returns TopicDeregistrationSuccess|TopicDeregistrationError {
+        // implement action to execute on topic-deregistration
+        }
+
+    remote function onUpdateMessage(UpdateMessage msg) returns Acknowledgement|UpdateMessageError {
+        // implement action to execute on content-update for topic
+        }
+
+    remote function onSubscription(Subscription msg) returns SubscriptionAccepted|SubscriptionPermanentRedirect|
+        SubscriptionTemporaryRedirect|BadSubscriptionError|InternalSubscriptionError {
+        // implement action to execute on new subscription
+        }
+
+    remote function onSubscriptionValidation(Subscription msg) returns SubscriptionDeniedError? {
+        // implement action to execute on subscription validation
+        }
+
+    remote function onSubscriptionIntentVerified(VerifiedSubscription msg) {
+        // implement action to execute on subscription intent verification
+        }
+
+    remote function onUnsubscription(Unsubscription msg) returns UnsubscriptionAccepted|BadUnsubscriptionError|
+        InternalUnsubscriptionError {
+        // implement action to execute on unsubscription
+        }
+
+    remote function onUnsubscriptionValidation(Unsubscription msg) returns UnsubscriptionDeniedError? {
+        // implement action to execute on unsubscription validation
+        }
+
+    remote function onUnsubscriptionIntentVerified(VerifiedUnsubscription msg) {
+        // implement action to execute on unsubscription intent verification
+        }
+    }    
+    ```
+
+**Hub Client Implementation**
+
+- The `websubhub:Client` is introduced to distribute the updated content to subscribers.
+- The example below is a sample use-case of the WebSub Hub Client.
+
+```ballerina
+    service / websubhubon hubListener {
```

Comment:
There are multiple issues in this sample? Shall we check and fix? @ayeshLK  Please see comments in-line.

```suggestion
    service / websubhub on hubListener {
```

Reply:
@ayeshLK

Reply:
```suggestion
    service /websubhub on hubListener {
```